sklearn/ensemble/_hist_gradient_boosting/tests/test_warm_start.py

```diff
@@ -99,8 +100,8 @@ def test_warm_start_early_stopping(GradientBoosting, X, y):
 
     n_iter_no_change = 5
     gb = GradientBoosting(
-        n_iter_no_change=n_iter_no_change, max_iter=10000,
-        random_state=42, warm_start=True, tol=1e-3
+        n_iter_no_change=n_iter_no_change, max_iter=200,
+        random_state=42, warm_start=True, tol=1e-3, early_stopping=False
```

Comment:
and here?

Reply:
early_stopping should explicitly be True for this test